opentelemetry-api/src/opentelemetry/resources/__init__.py

```diff
@@ -11,3 +11,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import Any, Dict, Union
+
+
+class Resource:
+    def __init__(self, labels: Dict[str, str]):
+        """
+        Construct a resource. Direct calling of the
```

Comment:
Why the short lines in docstrings?

Reply:
just a personal preference, but I agree that it is funky considering the linter.

I will reformat to match the pylint 89 character limit.